h2o-algos/src/main/java/hex/ensemble/Metalearner.java

```diff
@@ -116,6 +132,12 @@ protected void setCommonParams(P parms) {
     parms._max_runtime_secs = _maxRuntimeSecs;
     parms._weights_column = _model._parms._weights_column;
     parms._offset_column = _model._parms._offset_column;
+
+    // Checking if it was set in "metalearner_params" to keep backward compatibility
```

Comment:
instead of using reflection it would be better to introduce an API on model Parameters for setting early stopping criteria

in tree-based you have score_tree_interval, in GLM score_iteration_interval 

we could eg. add a method setScoreIterationInteval to Model.Parameters that would set the parameter if it is suppored